www.openfisca.fr/www.openfisca.fr.conf

```diff
@@ -0,0 +1,63 @@
+server {
+	listen 80;
+	server_name openfisca.fr;
+
+        location /.well-known {
+            root /tmp/renew-webroot;
+        }
+
+	location / {
+		return 301 https://www.$server_name$request_uri;
+	}
+}
+
+server {
+	listen 80;
+	server_name www.openfisca.fr;
+
+        location /.well-known {
+            root /tmp/renew-webroot;
+        }
+
+	location / {
+		return 301 https://$server_name$request_uri;
+	}
+}
+
+server {
+	listen 443 ssl;
+	server_name openfisca.fr;
+
+        ssl_certificate /etc/letsencrypt/live/openfisca.fr/fullchain.pem;
+        ssl_certificate_key /etc/letsencrypt/live/openfisca.fr/privkey.pem;
+
+	location / {
+		return 301 https://www.$server_name$request_uri;
+	}
+}
+
+server {
+	listen 443 ssl;
+	server_name www.openfisca.fr;
+
+        ssl_certificate /etc/letsencrypt/live/www.openfisca.fr-0001/fullchain.pem;
+        ssl_certificate_key /etc/letsencrypt/live/www.openfisca.fr-0001/privkey.pem;
+
+	access_log /var/log/nginx/www.openfisca.fr-access.log;
+	error_log /var/log/nginx/www.openfisca.fr-error.log;
+
+	location / {
```

Comment:
Why isn't there à 301 redirect here ?

Reply:
It redirects from http to httpS.

Reply:
All the other block mean to redirect the requests coming to `http://openfisca.fr`, `http://www.openfisca.fr` and `https://openfisca.fr` towards `https://www.openfisca.fr`.

This block indicates where `https://www.openfisca.fr` should actually find the application (`http://localhost:2010`).